app/src/androidTest/java/org/mozilla/fenix/ui/robots/ThreeDotMenuRobot.kt

```diff
@@ -8,21 +8,27 @@ package org.mozilla.fenix.ui.robots
 
 import androidx.test.espresso.Espresso.onView
 import androidx.test.espresso.assertion.ViewAssertions
+import androidx.test.espresso.assertion.ViewAssertions.matches
 import androidx.test.espresso.matcher.ViewMatchers
-import androidx.test.espresso.matcher.ViewMatchers.withText
+import androidx.test.espresso.matcher.ViewMatchers.*
 import androidx.test.platform.app.InstrumentationRegistry
 import androidx.test.uiautomator.UiDevice
 import org.hamcrest.Matchers.allOf
 import org.mozilla.fenix.R
 import org.mozilla.fenix.helpers.click
 
+
 /**
  * Implementation of Robot Pattern for the three dot (main) menu.
  */
 class ThreeDotMenuRobot {
     fun verifySettingsButton() = assertSettingsButton()
     fun verifyLibraryButton() = assertLibraryButton()
     fun verifyHelpButton() = assertHelpButton()
+    fun verifyThreeDotMenuExists() = threeDotMenuRecyclerViewExists()
+    fun verifyForwardButton() = assertForwardButton()
+    fun verifyBackButton() = assertBackButton()
+    fun verifyRefreshButton() = assertRefreshButton()
```

Comment:
I'm not sure I understand why these are all effectively aliases for other methods. Shouldn't these other methods be renamed, instead of forwarded to?

Reply:
@rpappalax what was our reasoning for this again?

Reply:
@AaronMT the convention some people use with espresso is that the person writing the test itself will do so using natural (user-centric) language.  Sometimes these tests are even written by non-technical people. In the robot, however, the functions defined are performing a series of actions behind the scenes to complete one user-centric activity.  In other words, we'd like our Espresso test to look almost verbatim like the test description we have in TestRail for manual testers for consistency.  We don't "have to" follow this convention, but is very useful for us to maintain a clean separation of concerns.